weejson/src/main/scala/com/rallyhealth/weejson/v1/Value.scala

```diff
@@ -215,7 +215,7 @@ object Obj {
   implicit def from(items: TraversableOnce[(String, Value)]): Obj = {
     val initialCapacity = items match {
       case is: mutable.IndexedSeq[_] => is.size
-      case _ => 2
+      case _                         => 2
```

Comment:
not that it's going to affect anything, but this whitespace change is a bit nutty.

Reply:
removed